### DISCUSSION TITLE
MonoGame.Content.Builder.Task.targets error in old and new projects

### DISCUSSION THREAD

**Discussion by Skahir** [MonoGame/MonoGame#8434] [General]
Hi all,

I've been working on and off on this monogame project and I'm running into a problem. I wanted to continue working on this old project but couldn't because of a build error.
```The command "dotnet "C:\Users\nando\.dotnet\tools\mgcb-editor.exe" /quiet /@:"C:\Users\nando\source\repos\Project1\Content\Content.mgcb" /platform:DesktopGL /outputDir:"C:/Users/nando/source/repos/Project1/Content/bin/DesktopGL/Content" /intermediateDir:"C:/Users/nando/source/repos/Project1/Content/obj/DesktopGL/net6.0/Content" /workingDir:"C:/Users/nando/source/repos/Project1/Content/"" exited with code -2147450749.	Project1	C:\Users\nando\.nuget\packages\monogame.content.builder.task\3.8.1.303\build\MonoGame.Content.Builder.Task.targets```

Naturally I tried creating a new project but as you can see this error also occurs in there. Does anybody know what is going wrong? 

I've been trying to fix it now several times and eventually caved and made this post. If anyone can help or point me in a direction that would be great. Thanks

**Comment by KelsamGames** [comment]
Open your Content.mgcb file with the MGCB editor and manually build your content. You will get errors in the UI that you can address. You should be able to do this by simply double-clicking your Content.mgcb file in Visual Studio, but if you have a config issue you may need to open it manually (if so, look in `C:\Users\nando\.nuget\` or `C:\Users\nando\.dotnet\` and search for `mgcb-editor-windows.exe`).

**Reply by Skahir** [reply]
When I build in the MGCB editor it has no problems but in vs22 it still has the same error.

Below is what the MGCB editor looks like after the build.
![image](https://github.com/user-attachments/assets/a95f56d4-1b22-4bda-bb41-98cf84ca3837)

This is what the output terminal says after I try to build in vs22:
```Build started at 19:00...
1>------ Build started: Project: Project1, Configuration: Debug Any CPU ------
1>Tool 'dotnet-mgcb' (version '3.8.1.303') was restored. Available commands: mgcb
1>Tool 'dotnet-mgcb-editor' (version '3.8.1.303') was restored. Available commands: mgcb-editor
1>Tool 'dotnet-mgcb-editor-linux' (version '3.8.1.303') was restored. Available commands: mgcb-editor-linux
1>Tool 'dotnet-mgcb-editor-windows' (version '3.8.1.303') was restored. Available commands: mgcb-editor-windows
1>Tool 'dotnet-mgcb-editor-mac' (version '3.8.1.303') was restored. Available commands: mgcb-editor-mac
1>
1>Restore was successful.
1>A fatal error was encountered. The library 'hostpolicy.dll' required to execute the application was not found in 'C:\Users\nando\.dotnet\tools\'.
1>Failed to run as a self-contained app.
1>  - The application was run as a self-contained app because 'C:\Users\nando\.dotnet\tools\mgcb-editor.runtimeconfig.json' was not found.
1>  - If this should be a framework-dependent app, add the 'C:\Users\nando\.dotnet\tools\mgcb-editor.runtimeconfig.json' file and specify the appropriate framework.
1>C:\Users\nando\.nuget\packages\monogame.content.builder.task\3.8.1.303\build\MonoGame.Content.Builder.Task.targets(142,5): error MSB3073: The command "dotnet "C:\Users\nando\.dotnet\tools\mgcb-editor.exe" /quiet /@:"C:\Users\nando\source\repos\Project1\Content\Content.mgcb" /platform:DesktopGL /outputDir:"C:/Users/nando/source/repos/Project1/Content/bin/DesktopGL/Content" /intermediateDir:"C:/Users/nando/source/repos/Project1/Content/obj/DesktopGL/net8.0/Content" /workingDir:"C:/Users/nando/source/repos/Project1/Content/"" exited with code -2147450749.
1>Done building project "Project1.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========```

**Comment by Skahir** [comment]
Found this solution in an earlier post and this fixed it. https://community.monogame.net/t/the-new-project-cannot-build/19152/3
